nulog-inference-service/start-nulog-inference.py

```diff
@@ -376,17 +345,19 @@ async def schedule_update_pretrain_model(logs_queue):
     loop.run_until_complete(task)
 
     if IS_CONTROL_PLANE_SERVICE:
-        init_model_task = loop.create_task(get_pretrain_model())
-        loop.run_until_complete(init_model_task)
+        init_model_task = loop.create_task(get_pretrain_model(logs_queue))
+        model_loaded = loop.run_until_complete(init_model_task)
+        if not model_loaded:
+            os._exit(1)
```

Comment:
I suggest `sys.exit()` over `os._exit()`. any reason why you went with the latter?

Reply:
Only because it lines up with what we do in go.  I have no strong preference.